rules/windows/defense_evasion_clearing_windows_event_logs.toml

```diff
@@ -22,9 +22,9 @@ tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 type = "query"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:wevtutil.exe and process.args:cl or
-  process.name:powershell.exe and process.args:Clear-EventLog
+event.category:process and event.type:(process_started or start) and 
+(process.name:wevtutil.exe or process.pe.original_file_name:wevtutil.exe) and 
+process.args:("/e:false" or cl or clear-log) or process.name:powershell.exe and process.args:Clear-EventLog
```

Comment:
```suggestion
event.category:process and event.type:(process_started or start) and 
(process.name:"wevtutil.exe" or process.pe.original_file_name:"wevtutil.exe") and 
process.args:("/e:false" or cl or "clear-log") or process.name:"powershell.exe" and process.args:"Clear-EventLog"
```

Hey @Samirbous , I receive a Lucene syntax warning with the current KQL query.  It does look like the query is accurate, but we probably want to strive to not trigger any warnings for users that may cause confusion. It looks like some of the fields need quotes around them (keywords with special chars and process/file name. 

![image](https://user-images.githubusercontent.com/57736958/100786172-5b703e80-33d7-11eb-8096-9a5f45044cd3.png)